servicepulse/install-servicepulse-in-iis.md

```diff
@@ -3,28 +3,92 @@ title: Install ServicePulse in IIS
 summary: Describes how to manually install ServicePulse in IIS.
 tags:
 - ServicePulse
+- Security
+- IIS
 ---
 
+### Prerequisites
+
+These instructions assume the following:
+
+* ServiceControl has installed and listening on http://localhost:33333/api
+* ServicePulse has been installed
+
+### Basic Setup
 [ServicePulse](introduction-and-installing-servicepulse.md), by default, is installed as a Windows Service that will self-host the ServicePulse web application.
 
 It is possible to manually install ServicePulse using IIS following these steps:
 
 * Extract ServicePulse files using, at a command prompt, the following command:
-* 
+
 ```
-ServicePulse.Host.exe` --extract --serviceControlUrl="http://localhost:8080/api" --outPath="C:\temp\SP"
+ServicePulse.Host.exe` --extract --serviceControlUrl="http://localhost:33333/api" --outPath="C:\ServicePulse"
 ```
 
 Note: `ServicePulse.Host.exe` can be found in the ServicePulse installation directory, whose default is `%programfiles(x86)%\Particular Software\ServicePulse`
 
 Once all the ServicePulse files are successfully extracted you can configure a new IIS web site whose physical path points to the location where files have been extracted.
 
-### Changing ServicePulse port
+When using IIS to host ServicePulse the ServicePulse.Host service is not used.  To remove the service uninstall ServicePulse from Add/Remove programs
+
+### Advanced Configuration
+
+ServicePulse relies on the ServiceControl REST API.  It is possible to add a reverse proxy to the web site using  the Microsoft [Application Request Routing](http://www.iis.net/downloads/microsoft/application-request-routing) IIS extension.
+This is useful if you which to lock down access to ServicePulse or if wish to expose the web site over a single port.
+
+Note: Please review the Security section below prior to carrying out the installation
+
+Installation Steps:
+
+1. Install the IIS ARR 3 extension (http://www.iis.net/downloads/microsoft/application-request-routing)
```

Comment:
make this an inline link 

    [IIS ARR 3 extension](http://www.iis.net/downloads/microsoft/application-request-routing)